installer/PowerToysSetup/Resources.wxs

```diff
@@ -11,7 +11,7 @@
   <Fragment>
     <!-- Resource directories should be added only if the installer is built on the build farm -->
     <?ifdef env.IsPipeline?>
-      <?foreach ParentDirectory in LauncherInstallFolder;FancyZonesInstallFolder;ImageResizerInstallFolder;ColorPickerInstallFolder;FileExplorerPreviewInstallFolder;HistoryPluginFolder;CalculatorPluginFolder;FolderPluginFolder;ProgramPluginFolder;ShellPluginFolder;IndexerPluginFolder;UnitConverterPluginFolder;ValueGeneratorPluginFolder;UriPluginFolder;WindowWalkerPluginFolder;OneNotePluginFolder;RegistryPluginFolder;VSCodeWorkspacesPluginFolder;ServicePluginFolder;SystemPluginFolder;TimeDatePluginFolder;WindowsSettingsPluginFolder;WindowsTerminalPluginFolder;WebSearchPluginFolder;PowerToysPluginFolder?>
+      <?foreach ParentDirectory in LauncherInstallFolder;FancyZonesInstallFolder;ImageResizerInstallFolder;ColorPickerInstallFolder;FileExplorerPreviewInstallFolder;HistoryPluginFolder;CalculatorPluginFolder;FolderPluginFolder;ProgramPluginFolder;ShellPluginFolder;IndexerPluginFolder;UnitConverterPluginFolder;ValueGeneratorPluginFolder;UriPluginFolder;WindowWalkerPluginFolder;OneNotePluginFolder;RegistryPluginFolder;VSCodeWorkspacesPluginFolder;ServicePluginFolder;SystemPluginFolder;TimeDatePluginFolder;WindowsSettingsPluginFolder;WindowsTerminalPluginFolder;WebSearchPluginFolder;PowerToysPluginFolder;ValueGeneratorPluginFolder?>
```

Comment:
`ValueGeneratorPluginFolder` was already in this list (between UnitConverterPluginFolder and UriPluginFolder).

Reply:
I thought I added the plugin to the installer a few commits ago, but it seems I missed a lot of things. Thank you for fixing that @jaimecbernardo!
Is there any documentation on how to do this so I can do this right before requesting a review next time? When I built it the powertoys solution and the installer locally the plugin was installed by default.

Reply:
We should order the lists alphabetical

Reply:
Sorry, this indeed ended up failing release CI :) Thanks for noticing it.
Regarding documentation, we'd definitely benefit from it. But it's also a moving target, https://github.com/microsoft/PowerToys/pull/27451 will change it again, for example.